crates/ruff_linter/src/rules/airflow/rules/moved_to_provider_in_3.rs

```diff
@@ -590,15 +586,13 @@ fn check_names_moved_to_provider(checker: &Checker, expr: &Expr, ranged: TextRan
         },
 
         // apache-airflow-providers-cncf-kubernetes
-        ["airflow", "executors", "kubernetes_executor_types", "ALL_NAMESPACES"] => Replacement::ImportPathMoved{
-                original_path: "airflow.executors.kubernetes_executor_types.ALL_NAMESPACES",
-                new_path: "airflow.providers.cncf.kubernetes.executors.kubernetes_executor_types.ALL_NAMESPACES",
+        ["airflow", "executors", "kubernetes_executor_types", "ALL_NAMESPACES"] => Replacement::ProviderName{
+                name: "airflow.providers.cncf.kubernetes.executors.kubernetes_executor_types.ALL_NAMESPACES",
             provider: "cncf-kubernetes",
             version: "7.4.0"
```

Comment:
nit: Is this how `rustfmt` is formatting this with `name` field being indented more than the other fields? Seems like a bug if that's the case.

Reply:
hmmm... didn't notice this one. let me check again

Reply:
I guess this was just my mistake. just fixed it. thanks!